src/export/StructureDefinitionExporter.ts

```diff
@@ -541,6 +547,58 @@ export class StructureDefinitionExporter implements Fishable {
     });
   }
 
+  private setCharacteristics(structDef: StructureDefinition, fshDefinition: Logical) {
+    // characteristics are set using the structuredefinition-type-characteristics extension
+    // http://hl7.org/fhir/StructureDefinition/structuredefinition-type-characteristics
+    // the allowed codes are in the type-characteristics-code value set.
+    // http://hl7.org/fhir/ValueSet/type-characteristics-code
+    // the value set is composed of all codes in the type-characteristics-code code system.
+    // http://hl7.org/fhir/type-characteristics-code
+    // however, the can-be-target code may be missing. if so, use the logical-target extension:
+    // http://hl7.org/fhir/tools/StructureDefinition/logical-target
+    // see this thread for a discussion of this:
+    // https://chat.fhir.org/#narrow/stream/179177-conformance/topic/Reference.20to.20Logical.20Model/near/358606109
+    if (this.typeCharacteristicCodes == null) {
+      logger.warn(
+        `Type characteristics code system not found. ${fshDefinition.name} may include invalid characteristics.`,
```

Comment:
I think people might worry about this message as it is worded now (e.g., "what did it see that made it think these codes may be invalid?"). Perhaps it would be better to say something like this?
```suggestion
        `Type characteristics code system not found. Skipping validation of characteristics for ${fshDefinition.name}.`,
```